go-fuzz-build/main.go

```diff
@@ -580,3 +625,46 @@ func main() {
 	dep.Main(target.%v)
 }
 `
+
+var mainSrcLibFuzzer = `
```

Comment:
I will probably rewrite these to use text/template at some point in the future, since they are pretty large, and it is hard to easily see the formatting directives. Fine for now, though.